dts/x86/intel/ia32.dtsi

```diff
@@ -75,8 +76,7 @@
 			interrupt-parent = <&intc>;
 			alarms-count = <1>;
 
-			status = "okay";
+			status = "disabled";
```

Comment:
This can likely also be added in the testsuite in a board overlay, maybe with a descriptive comment, in order to avoid breaking unrelated things.

Reply:
I don't think this should be in the test suite, I don't know what criteria we use to determine if a node is enabled in the dtsi file, but for an RTC, I would expect it to be disabled by default

Reply:
Removed this change from the PR